equinox-platformx/react-native-airplay-btn
# update deprecated notification

fix: update deprecated MPVolumeViewWirelessRouteActiveDidChangeNotification

To test you can point the app with this branch  in package.json.
HOW TO TEST:
1-
start a video class (live or not)
start airplaying
check that the background image appears in the main device

2-
start mirroring
start a video class
check that the background image appears in the main device

[MD-2822] [MD-2836]

[MD-2822]: https://platformx.atlassian.net/browse/MD-2822
[MD-2836]: https://platformx.atlassian.net/browse/MD-2836
